assets/javascripts/kitten/components/navigation/horizontal-nav.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react'
+import classNames from 'classnames'
+import Markdown from 'react-markdown'
+
+export class HorizontalNav extends Component {
+  renderItems() {
+    const { items } = this.props
+
+    return items.map((item, key) => this.renderItem(item, key))
```

Comment:
Est-ce qu'on a commencé à gérer les clés autrement sur les composants ? À savoir ne pas utiliser l'index d'un `map` (cf. https://reactjs.org/docs/lists-and-keys.html#keys) pour une `key` ? On pourrait utiliser le `text` et l'incrémenter si c'est le même ?

Reply:
Tu devais être saoule Fanny pour écrire ça !

Reply:
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Tu peux t'inspirer de https://github.com/KissKissBankBank/kitten/blob/master/assets/javascripts/kitten/components/lists/tag-list.js par exemple. La `key` se trouve directement l'item.

```js
items=[
  { key: 'foo', name: 'Foo', … },
  { key: 'bar', name: 'Bar', … }
]
```